scripts/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM squidfunk/mkdocs-material:8.5.9
+
+# ENV mike version 2 is required in order to enable ENV vars for the container. By the time mike 2 is released we'd use a specific dev version
+RUN pip install "mike>=2" || pip install -e git+https://github.com/jimporter/mike.git@2faa33ca4cf0da62c9c0ac7fbe8b60688733d7f1#egg=mike
```

Comment:
This is fine for me, but I'd rather just use the commit hash version. This way we might experience issues or different behaviour in future once version 2 is released.